src/types/radio/radio.html

```diff
@@ -1,8 +1,10 @@
-<md-radio-group ng-model="model[options.key]" md-theme="{{to.theme}}">
+<md-radio-group ng-model="model[options.key]" md-theme="{{to.theme}}" ng-disabled="to.disabled">
     <md-radio-button
-            ng-repeat="option in to.options"
-            ng-disabled="to.disabled"
-            ng-value="option[to.valueProp || 'value']">
-            {{option[to.labelProp || 'name']}}
+        ng-repeat="option in to.options track by $index"
+        aria-label="{{to.options[$index].ariaLabel}}"
+        class="{{to.options[$index].className}}"
+        ng-disabled="to.options[$index].disabled || to.disabled"
+        ng-value="to.options[$index].value">
```

Comment:
I would add the possibility of define the value property name.
Like this `ng-value="to.options[$index][to,valueProp || 'value']"`.

Same goes to the next line,
~~`{{to.options[$index].name`~~
replaced by
`{{to.options[$index][to.labelProp || 'name']`

What do you say?